### ISSUE TITLE
Als [] wil ik een separate Contacten API als standaard

### ISSUE THREAD

**Issue by Hugo-ter-Doest** [VNG-Realisatie/gemma-zaken#1225]
...zodat contactregistratie niet meer onderdeel is van de Zaken API en contactregistratie onafhankelijk van de Zaken API kan worden gebouwd en uitgerold.

Toelichting:
Contactregistratie is een activiteit die onafhankelijk van zaakgericht werken plaatsvindt. Een contact kan tot een zaak leiden, een contact kan op een later moment aan een zaak worden gerelateerd, en er kunnen contactmomenten zijn n.a.v. een zaak.

Onderstaande "vink-lijst" is voor het team zelf om in te vullen en hoeft niet door de indiender te worden ingevuld.

**Bepaling prioriteit door PO**

- [ ] verbreding of verdieping API's
- [ ] stimuleert gebruik door gemeenten
- [ ] stimuleert gebruik door leveranciers

... eventueel nog toelichting door PO

**Definition of ready**
- [ ] Iedereen in het team begrijpt de user story
- [ ] de gewenste (aanvulling op de) functionaliteit van de API's duidelijk en beschreven is.
- [ ] Is klein genoeg (maximaal 1/5 van sprint)
- [ ] Product Owner akkoord en voorzien van prioriteit (mag alleen afgevinkt worden door PO)
- [ ] Idee hebben van hoe deze user story kan worden gedemonstreerd.
- [ ] _Globale_ oplossingsrichting bekend
- [ ] Vastgelegd in Github en geplaatst in kolom ready

**Definition of done**
- [ ] Er is een OAS 3.0 specificatie
- [ ] Er is een referentieimplementatie
- [ ] Er zijn tests(cases) aanwezig die de wijziging aantonen en waarmee de user story getest kan worden.
- [ ] De technische specificatie (standaard.md) is gepubliceerd leesbaar
- [ ] Gebruikte gegevensmodel is na **iedere sprint** bijgewerkt.

**Acceptatiecriteria**
- [ ] De DSO URI- en API-strategie worden gevolgd of afwijkingen zijn vastgelegd als ontwerp keuze
- [ ] Er zijn geen bekende GEMMA tegenstrijdigheden **of** afwijkingen zijn vastgelegd.

**Taken**
- [ ] Implementeren in referentie-implementatie [_verantwoordelijke_]
- [ ] Schrijven (unit) test voor referentie-implementatie [_verantwoordelijke_]
- [ ] Genereren/opstellen van OAS 3.0 [_verantwoordelijke_]
- [ ] Human Readable publiceren Open API Specificatie (v.3.0) [_verantwoordelijke_]
- [ ] Documentatie bijwerken
- [ ] Gegevensmodel bijwerken

**Comment by adgerrits** [comment]
Is eerder over gediscussieerd en toen NIET opgenomen binnen de [GEMMA informatiearchitectuur](https://www.gemmaonline.nl/index.php/GEMMA_2_Applicatiearchitectuur) als generieke functionaliteit. Kijkend naar deze ontwikkeling lijkt het me logisch dat dit bij update van de GEMMA-IA nu wel gebeurd (@jeffreygortmaker1 ?)

**Comment by jeffreygortmaker1** [comment]
ja. ZGW in Gegevenslandschap (https://www.gemmaonline.nl/images/gemmaonline/f/f6/20190620_-_Zaakgericht_werken_in_het_Gemeentelijk_Gegevenslandschap_v101.pdf) benoemt dit ook: Klanten en -contactenAPI. Mogen er ook 2 worden.
Hebben we wel ene informatiemodelletje klanten en klantcontacten/contactmomenten/.. nodig.

**Comment by sergei-maertens** [comment]
XL ingeschat wegens mogelijke link met personen etc.

**Comment by joeribekker** [comment]
Iemand (@jeffreygortmaker1, @adgerrits ) moet ons even uitleg geven over de business case en logica van klantcontacten en contacten, voordat we met deze User Story beginnen.

**Comment by adgerrits** [comment]
Alvast een voorzet hoe de werkelijkheid er volgens mij uit ziet: 
Klant (persoon of bedrijf) M:N Contactmoment M:N Zaak M:N Registratie
M en N kunnen zijn: 0, 1 of meerdere.

**Comment by EdwinCoster** [comment]
Deze user story leidt tot een wijziging van het gegevensmodel.

**Comment by michielverhoef** [comment]
Volgens mij is deze wens ingevuld met Contactmomenten API en Klanten API?
@hdksi kun jij hier iets van zeggen?
